lib/Dialect/SPIRV/SPIRVOps.cpp

```diff
@@ -154,6 +154,43 @@ printMemoryAccessAttribute(LoadStoreOpTy loadStoreOp, OpAsmPrinter &printer,
   elidedAttrs.push_back(spirv::attributeName<spirv::StorageClass>());
 }
 
+template <typename CastOp>
```

Comment:
Do we need a template here? I think just using a `Operation *op` should be fine here?